### PR TITLE
default flags for cluster runs

### PR DESCRIPTION
**Pull request by VictorGillioz**
PR to change the default flag values to work on the cluster.

### CODE REVIEW

File: meshnets/utils/model_training.py [hunk 1]
```diff
@@ -123,4 +123,4 @@ def train_model(config, experiment_config, train_dataset, validation_dataset):
 
     trainer.fit(lightning_wrapper, train_loader, validation_loader)
 
-    return lightning_wrapper
+    #return lightning_wrapper
```
Comment (AugustoPeres) [inline meshnets/utils/model_training.py:126]: Remove commented code

**Review by AugustoPeres — APPROVED** [review]

File: meshnets/tune.py [hunk 2]
```diff
@@ -30,20 +30,20 @@
...
 
 # Dataloaders flags
-flags.DEFINE_multi_integer('batch_size', [1, 2, 4], 'The batch size.')
+flags.DEFINE_multi_integer('batch_size', [4, 8], 'The batch size.')
 flags.DEFINE_integer('num_workers_loader', 2,
                      'The number of workers for the data loaders.')
 
 # Model parameters flags
-flags.DEFINE_list('latent_size', [8, 16, 32],
+flags.DEFINE_list('latent_size', [8, 32, 64],
```
Comment (fabiocruz) [inline meshnets/tune.py:38]: My OCD tells me 16 should be in this list 😅

File: meshnets/tune.py [hunk 3]
```diff
@@ -54,12 +54,12 @@
 
 # Strategy flags
 flags.DEFINE_integer('num_workers_ray', 2, 'The number of workers.')
-flags.DEFINE_integer('num_cpus_per_worker', 1,
+flags.DEFINE_integer('num_cpus_per_worker', 12,
                      'The number of cpus for each worker.')
-flags.DEFINE_bool('use_gpu', False, 'Whether to use gpu or not.')
+flags.DEFINE_float('num_gpus_per_worker', 0.5, 'The number of gpus for each worker.')
```
Comment (fabiocruz) [inline meshnets/tune.py:59]: Add something like "If set to zero, GPUs are not used."

**Review by fabiocruz — APPROVED** [review]
LGTM!

File: meshnets/tune.py (hunk 2 above)
Comment (VictorGillioz) [reply meshnets/tune.py:38]: Added it

**Pull request merged**